cmd/list_test.go

```diff
@@ -22,30 +22,6 @@ import (
 	"github.com/appsody/appsody/cmd/cmdtest"
 )
 
-func TestList(t *testing.T) {
```

Comment:
This test runs an `appsody list` after adding a repo with a v1 index. Since we're no longer using this format, I initially updated the test to use the newer v2 index but found that there's already a test that handles this case. So, I deleted the test altogether especially since this PR makes it so that users can't add a v1 index at all.